### PR TITLE
chore(web): Story block edit Panel layout

### PR DESCRIPTION
**Pull request by mkumbobeaty**
# Overview

## What I've done

- Update Story block edit Panel layout

### CODE REVIEW

File: web/src/services/i18n/translations/en.yml [hunk 1]
```diff
@@ -257,6 +257,7 @@ Spacing settings: Spacing settings
 New Camera: New Camera
 New Camera Button: New Camera Button
 Buttons List: Buttons List
+Button Title: 'Button Title'
```
Comment (KaWaite) [inline web/src/services/i18n/translations/en.yml:260]:
```suggestion
Button Title: Button Title
```

File: web/src/services/i18n/translations/ja.yml [hunk 2]
```diff
@@ -236,6 +236,7 @@ Spacing settings: Spacing設定
 New Camera: 新しいカメラ
 New Camera Button: 新カメラボタン
 Buttons List: ボタン一覧
+Button Title: ''
```
Comment (KaWaite) [inline web/src/services/i18n/translations/ja.yml:239]:
```suggestion
Button Title: タイトル
```

File: web/src/beta/lib/core/StoryPanel/Block/builtin/Layer/Editor/index.tsx [hunk 3]
```diff
@@ -67,35 +67,40 @@ const CameraBlockEditor: React.FC<Props> = ({
...
+          addItem={handleItemAdd}
+          removeItem={handleItemRemove}
+          onItemDrop={handleItemMove}
+          selected={selected}
+          onSelect={setSelected}
+          atLeastOneItem
         />
+        <FieldGroup disabled={!editorProperties}>
+          <TextField
+            name={t("Button Title")}
```
Comment (KaWaite) [inline web/src/beta/lib/core/StoryPanel/Block/builtin/Layer/Editor/index.tsx:83]: Does this property not have a title (`editorProperties?.title?.title`)? If it doesn't, it should, so its better to add to the manifest than make this a unique case on the frontend

**Review by KaWaite — APPROVED** [review]

**Review by pyshx — APPROVED** [review]

File: web/src/beta/lib/core/StoryPanel/Block/builtin/Layer/Editor/index.tsx (hunk 3 above)
Comment (mkumbobeaty) [reply web/src/beta/lib/core/StoryPanel/Block/builtin/Layer/Editor/index.tsx]: there is the title so I added it

**Pull request merged**